n2y/plugins/linktopage.py

```diff
@@ -0,0 +1,38 @@
+import logging
+
+from n2y.blocks import Block
+
+logger = logging.getLogger(__name__)
+
+class LinkToPage(Block):
```

Comment:
The plugin class needs to inherit from the existing `n2y.blocks. LinkToPageBlock` class in order for this to work. (This could be made more explicit, but this is what's discussed in bullet point 2 in the README here https://github.com/innolitics/n2y#plugins )

Reply:
I would have thought you'd get an exception raised from here because this is not subclassing `n2y.blocks.LinkToPageBlock`:

https://github.com/innolitics/n2y/blob/main/n2y/notion.py#L149-L155

Were you seeing this exception? If not, this is a bug in itself.

Reply:
Yes, I do understand that the instructions were that the plugin should inherit from the LinkToPageBlock.  

I brought up some issues to Bimba concerning this inheritance.  I can elaborate.  Anyway, he suggested that I could just inherit from Block.  Sounded good to me, so as not to disturb the original functioning of the class for existing customers. I can explain to you.  The issue is described in my comments on files of previous commits. 

The only exception that I saw when trying to subclass n2y.blocks.LinkToPageBlock was that if the parent of that class were WarningBlock, then the interpreter complained that I wasn't overriding WarningBlock, as I recall.  I would need to recreate the error in order to tell you more.

Reply:
I see, yeah @bimbashrestha @idolcemia as written, I believe you must inherit from LinkToPageBlock. I don't think you should need to modify the `n2y.blocks.LinkToPageClass` at all. Your class will presumably override both the `__init__` and `to_pandoc` methods, thus removing any warning functionality that you don't want in your class. Since you won't be touching `n2y.blocks.LinkToPageClass` at all, you can be sure it won't affect other people using it.

I'm surprised that it would complain about you not overriding WarningBlock. That sounds like a bug in the error handling code. If you happen to reproduce it, please copy the stack trace and let me know.